templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,17 @@
+# Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button ![chevron](/static/images/help/chevron.png)
+to bring up the stream dropdown menu and press **Mute the stream**. After
+pressing, the stream should turn gray in the streams list.
+
+![stream dropdown](/static/images/help/stream-left-sidebar-color.png)
```

Comment:
You can add an image of the stream turning gray here instead of **stream dropdown** image. The stream dropdown image can be moved to the end of line 5.